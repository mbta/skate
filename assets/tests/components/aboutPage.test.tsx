import React from "react"
import renderer from "react-test-renderer"
import AboutPage from "../../src/components/aboutPage"

describe("About page", () => {
  test("renders", () => {
    const tree = renderer.create(<AboutPage />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
