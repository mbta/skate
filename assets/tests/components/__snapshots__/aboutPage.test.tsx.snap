// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders 1`] = `
<div
  className="c-page c-page--about"
>
  <div
    className="c-page__container"
  >
    <h1
      className="c-page__title"
    >
      About Skate
    </h1>
    <p>
      Skate is an active work-in-progress. In the coming months, you’ll notice new tools like shuttle tracking. You can also expect changes and refinements to the route ladders as we continue building.
    </p>
    <div
      className="c-page__section"
    >
      <h2
        className="c-page__header"
      >
        Device Support
      </h2>
      <p>
        Need tablet support? Call the MBTA Help Desk at x5761.
      </p>
    </div>
    <div
      className="c-page__section"
    >
      <h2
        className="c-page__header"
      >
        Frequently Asked Questions
      </h2>
      <p>
        Have questions about using Skate? We’re making updates all the time. Read the latest news and refresh your training by visiting our FAQ.
      </p>
      <p>
        <a
          className="c-page__link"
          href="https://docs.google.com/document/d/e/2PACX-1vQVzXUcQWg-nKCkb8CyModNUCXDHHRxgqPdtlkm1GQOK9aCrGMSJAoC2J8JuuGFPGnPkcetrZEk6ERH/pub"
          target="_blank"
        >
          FAQ
        </a>
      </p>
    </div>
    <div
      className="c-page__section"
    >
      <h2
        className="c-page__header"
      >
        Feedback
      </h2>
      <p>
        Have feedback, a suggestion, or need other support related to Skate? Use our feedback form and someone on our team will be in touch with you.
      </p>
      <p>
        <a
          className="c-page__link"
          href="https://form.asana.com?hash=1a2a65ebbc7a183eb3298889a2cc9ccaf1e95cfa7a6f9ca8f226dfdb8dbb4ab0&id=1129738905260896"
          target="_blank"
        >
          Give Feedback
        </a>
      </p>
    </div>
  </div>
</div>
`;
